integration-tests/bulkExportTestHelper.ts

```diff
@@ -80,8 +86,8 @@ export default class BulkExportTestHelper {
     }
 
     async getExportStatus(statusPollUrl: string, expectedSubstring = ''): Promise<any> {
-        const fiveMinuteFromNow = new Date(new Date().getTime() + this.FIVE_MINUTES_IN_MS);
-        while (new Date().getTime() < fiveMinuteFromNow.getTime()) {
+        const eightMinuteFromNow = new Date(new Date().getTime() + this.EIGHT_MINUTES_IN_MS);
```

Comment:
```suggestion
        const cuttOffTime = new Date(new Date().getTime() + this.EIGHT_MINUTES_IN_MS);
```

Sorry thought of an even better name that isn't aligned with a specific value of the variable